prow/plugins/releasenote/releasenote.go

```diff
@@ -81,11 +84,12 @@ func handle(gc githubClient, log *logrus.Entry, ic github.IssueCommentEvent) err
 		return nil
 	}
 
-	// Only allow authors and assignees to add labels.
-	isAssignee := ic.Issue.IsAssignee(ic.Comment.User.Login)
+	// Only allow authors and org members to add labels.
+	isMember, _ := gc.IsMember(trustedOrg, ic.Comment.User.Login)
```

Comment:
I don't think we should ignore the error.  If we can't verify membership, we shouldn't allow release note label to be set.